internal/tracker/source/tekton/tekton.go

```diff
@@ -405,6 +425,20 @@ func PreparePackage(i *PreparePackageInput) (*hub.Package, error) {
 		},
 	}
 
+	uniqueImages := make(map[string]bool)
+	for _, step := range steps {
+		if step.Image != "" {
+			if _, value := uniqueImages[step.Image]; !value {
```

Comment:
Can we rename `value` to `ok` or `found`? We are actually not using the value, so I think this may make the intent clearer.